spec/rails_helper.rb

```diff
@@ -65,5 +65,4 @@
   config.include RailsCompatibilitySpecHelper, type: :controller
 end
 
-ENV['MIXPANEL_TOKEN'] = 'fakemixpaneltoken'
```

Comment:
I'm surprised we can remove this, since the mixpanel analytics code blows up without it.

Reply:
we never actually call `MixpanelClient#track_assignment` in the specs, we always stub it out